spec/metric_fu/metrics/rails_best_practices/configuration_spec.rb

```diff
@@ -20,9 +20,23 @@
         end
       end
 
-      it "should set @rails_best_practices to {}" do
+      it "should default @rails_best_practices to { :silent => true }" do
         load_metric "rails_best_practices"
-        expect(MetricFu::Metric.get_metric(:rails_best_practices).run_options).to eql({})
+        expect(MetricFu::Metric.get_metric(:rails_best_practices).run_options).to eq(
+                                                                                    exclude: [],
+                                                                                    silent: true
+                                                                                  )
+      end
+
+      it "can configure @rails_best_practices 'exclude' using the sugar" do
+        load_metric "rails_best_practices"
+        rbp = MetricFu::Metric.get_metric(:rails_best_practices)
+        rbp.exclude = ["config/chef"]
```

Comment:
@bf4 I spent a couple minutes looking into the spec failures. The only thing I can think of is each of the Travis runs is running the specs in random order. Since they randomly seemed to fail, could it be that this line is at fault? If we set `.exclude` here, is that value carrying over to the other specs? In other words, do we know for sure if the metric has its default settings at the start of each spec?